code/game/machinery/doors/poddoor.dm

```diff
@@ -59,9 +59,28 @@
 /obj/machinery/door/poddoor/try_to_activate_door(mob/user)
  	return
 
-/obj/machinery/door/poddoor/try_to_crowbar(obj/item/I, mob/user)
+/obj/machinery/door/poddoor/crowbar_act(mob/user, obj/item/I)
+	if(user.a_intent == INTENT_HARM)
+		return
+	. = TRUE
+	if(operating)
+		return
+	if(!I.use_tool(src, user, 0, volume = 0))
```

Comment:
The entire purpose of this redefinition of this proc is to make sure the tool doesn't play a sound here, right? Could you instead define a variable like `play_tool_sound_on_force` on `/obj/machinery/door/poddoor/crowbar_act` that would control whether to use the tool volume or 0 for the `use_tool` volume?

Reply:
Now that I think bout it, I dont think any doors played the crowbar sound on being forced before the tool refactor

Ill just change the original door proc to not make a sound

Reply:
Fine by me